src/components/card/card.stories.mdx

```diff
@@ -73,6 +67,14 @@ The Twig template supports three main blocks (all optional):
   </Story>
 </Canvas>
 
+```twig
+{% embed '@cloudfour/components/card/card.twig' only %}
+  {% block heading %}Lorem ipsum dolor sit amet{% endblock %}
+  {% block content %}<p>Consectetur adipiscing elit...</p>{% endblock %}
+  {% block footer %}<p>{{'now'|date('M j, Y')}}</p>{% endblock %}
+{% endembed %}
+```
```

Comment:
It's totally fine to write this as static code in cases where it isn't feasible to automate, but I would suggest passing that along to the story parameters so our docs have a consistent experience of where to find code examples for a given story.

GitHub won't let me make a suggestion on unchanged code, but here's how you could move this chunk to a parameter:

```jsx
<Canvas>
  <Story
    name="Content Blocks"
    args={{ show: ['heading', 'content', 'footer'] }}
    parameters={{
      docs: {
        source: {
          code: `{% embed '@cloudfour/components/card/card.twig' only %}
  {% block heading %}Lorem ipsum dolor sit amet{% endblock %}
  {% block content %}<p>Consectetur adipiscing elit...</p>{% endblock %}
  {% block footer %}<p>{{'now'|date('M j, Y')}}</p>{% endblock %}
{% endembed %}`,
        },
      },
    }}
  >
    {singleDemoStory.bind({})}
  </Story>
</Canvas>
```

Reply:
OMG I feel so _silly_. 🤣 

I appreciate this, thanks so much, @tylersticka! 🌟

Reply:
No problem, Gerardo! And don't feel silly for not remembering or knowing about an option buried four levels deep in a largely-unused configuration object! 😅

Reply:
(BTW, you can also use `templateSource` instead of `source` and provide a function that accepts some parameters I can't remember, but one of those are the `args` you receive. So if you find yourself writing a lot of repetitive code, you could write a function earlier in the file that returns different examples depending on `args`. I think what you have here is fine and dandy, just thought I'd let you know!)

Reply:
Thanks, @tylersticka! 😄 

For my future self, is there documentation I can refer to somewhere for these properties? I still don't have a feel for where to look for when I run into these, what feels to me like magic, properties. 🔍 👀 😮

Reply:
Unfortunately, I've just found this stuff spelunking through various examples: https://github.com/cloudfour/cloudfour.com-patterns/issues/1004#issuecomment-852578742

Reply:
Ah, got it. Thank you! 🎓

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1315/commits/0553e6bb2060526c1aae18f09a365d67a86c7933 https://github.com/cloudfour/cloudfour.com-patterns/pull/1315/commits/a8885134cd0a18188d3589c83d130b068f19c1ed